lib/formtools/plugins/plugins-helpers.js

```diff
@@ -23,8 +23,8 @@ var defaults = function (options, isEmbeddedDocument) {
     }
 
     // dateCreated and dateModified will be hidden by default
```

Comment:
In the form dateCreated and dateModified fields are set to be hidden by default. So, I think the comment is correct that these two fields will be hidden by default in form.